doc/content/reference/application-packages/using-the-api.md

```diff
@@ -0,0 +1,262 @@
+---
+title: "Using the API"
```

Comment:
This will come up in search for "API" so change to "Managing Application Packages using the API"